### ISSUE TITLE
Update 2016-04-09 ~ Add JSON files & viewer

### ISSUE THREAD

**Issue by theo-armour** [ladybug-tools/3d-models#1]
![image](https://cloud.githubusercontent.com/assets/547626/14408480/9b67369e-feab-11e5-9552-459892821cbc.png)

---

Lots more work needed here.

---
### 2016-04-09
- Add Viewer
- Add Robie House in JSON
- Add Schroder House in JSON
### 2016-04-08
- First commit
- Add Urban Model in OBJ
- Add Urban Model in JSON

**Comment by ayezioro** [comment]
Hi,
Looks nice.
In the meantime some "bugs" in all models:
The houses (Robie, Schroeder) change scale when you change the parameters. They became too small for the sunpath.
Both urban models are rotated towards XZ plane. They also become bigger when you change any parameter.

Wishes:
To control the location of the project (Latitude/Longitude).
To have shadow fan (as i saw in one of the previous examples).
Start month from 1 instead 0. Day from 1 to 31 (instead of 0 to 31)

-A.

**Comment by theo-armour** [comment]
@ayezioro 

Thanks for the speedy look-see

> > In the meantime some "bugs" in all models:

Yes. all very R1. ;-)

Yup, the new 'auto-scale' on load is not yet being remembered.

And the urban models JSON certainly are rotated.

Actually all three models have numerous issues. Getting good models is not all all easy.

Generally there's no need to report bugs like these, as I see them just as easily as you do,

What you are looking at is the start to a light-weight viewer.

Most of the effort is here:

http://ladybug-analysis-tools.github.io/ladybug-web/

The current main editing tool - which covers your wishes - is here:

http://ladybug-analysis-tools.github.io/ladybug-web/shadow-opener/

The early shadow fan is here:

http://ladybug-analysis-tools.github.io/ladybug-web/multiple-lights/

It will be added to one of the viewers very soon.

Please note that this is all still at an early stage. I took my first crack at this on about March 5. So I still really don't know what I'm doing.

But things do seem to be settling down a bit. So I hope that before too long I can say: Hey, let's start hammering these scripts!

In the mean time you can just enjoy the progress. 

What I would really to see are online working examples  - or even just images - where these sorts of calculations are dome really well.

Again thanks for your good thoughts,

Theo

**Comment by ayezioro** [comment]
Hi,
I've just uploaded a file. It was modeled by one of my students. It is a Bauhaus style building from the 1930's.
Not sure i upload it right or at the right place, so let me know.
-A.

**Comment by theo-armour** [comment]
@ayezioro 

I heart Bauhaus.

Unfortunately the model doesn't load. And I moved it int the models folder - wjich I will eventually rename 'obj'.

There were some obvious errors which I tried to fix but it still did not load.

Two things to try.
1. If you export from Rhino (or wherever), the try importing the model back into Rhino
2. See if you can import the file into the [Three.js Editor](http://mrdoob.github.io/three.js/editor/)

In any case, thank you for trying. As you will learn, getting models is not easy.

**Comment by ayezioro** [comment]
Just uploaded the model as json.
I can see it in the Three.js editor.

-A.

**Comment by ayezioro** [comment]
Uploaded 3 examples of solar envelopes at the JSON folder.
-A.

**Comment by theo-armour** [comment]
@ayezioro 

The files sound interesting but unfortunately they are of a JSON type that I am not very familiar with. They appear to be binary-encoded STEP/IGES files. In any case they have no relation to the Three.js JSON we have readers for.

But, thank you for trying.

BTW, I found that the edits I made to your OBJ file actually did work. I am using it and will be showing it to you - in another post.
